carina-api/src/test/java/com/qaprosoft/apitools/validation/method/MockServer.java

```diff
@@ -0,0 +1,33 @@
+package com.qaprosoft.apitools.validation.method;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+public class MockServer {
+
+    private WireMockServer wireMockServer;
+
+    public MockServer() {
+        wireMockServer = new WireMockServer(wireMockConfig().port(8080));
```

Comment:
please move port to some project properties.
also try to use another port rather than 8080
since that one is rather popular and can be easily used by anything else.